src/system/math/Mtx.h

```diff
@@ -16,49 +16,75 @@ namespace Hmx {
         Vector3 z;
 
         // all of these are weak
-        Matrix3(){}
+        Matrix3() {}
 
-        Matrix3(const Matrix3& mtx){
-            x = mtx.x; y = mtx.y; z = mtx.z;
+        Matrix3(const Matrix3 &mtx) {
+            x = mtx.x;
+            y = mtx.y;
+            z = mtx.z;
         }
 
-        Matrix3(const Vector3& v1, const Vector3& v2, const Vector3& v3) : x(v1), y(v2), z(v3) {}
-
-        Matrix3(float f1, float f2, float f3, float f4, float f5, float f6, float f7, float f8, float f9) :
-            x(f1, f2, f3), y(f4, f5, f6), z(f7, f8, f9) {}
-
-        void Set(float f1, float f2, float f3, float f4, float f5, float f6, float f7, float f8, float f9){
-            x.Set(f1,f2,f3); y.Set(f4,f5,f6); z.Set(f7,f8,f9);
+        Matrix3(const Vector3 &v1, const Vector3 &v2, const Vector3 &v3)
+            : x(v1), y(v2), z(v3) {}
+
+        Matrix3(
+            float f1,
+            float f2,
+            float f3,
+            float f4,
+            float f5,
+            float f6,
+            float f7,
+            float f8,
+            float f9
+        )
+            : x(f1, f2, f3), y(f4, f5, f6), z(f7, f8, f9) {}
+
+        void
+        Set(float f1,
+            float f2,
+            float f3,
+            float f4,
+            float f5,
+            float f6,
+            float f7,
+            float f8,
+            float f9) {
+            x.Set(f1, f2, f3);
+            y.Set(f4, f5, f6);
+            z.Set(f7, f8, f9);
         }
```

Comment:
Probably should manually format these to save space and give it a pattern:
```suggestion
        // clang-format off
        Matrix3(
            float f1, float f2, float f3,
            float f4, float f5, float f6,
            float f7, float f8, float f9
        )
            : x(f1, f2, f3), y(f4, f5, f6), z(f7, f8, f9) {}

        void
        Set(
            float f1, float f2, float f3,
            float f4, float f5, float f6,
            float f7, float f8, float f9
        ) {
            x.Set(f1, f2, f3);
            y.Set(f4, f5, f6);
            z.Set(f7, f8, f9);
        }
        // clang-format on
```